source/Xunit.Gherkin.Quick/StepDefinition/BaseStepDefinitionAttribute.cs

```diff
@@ -6,7 +6,7 @@ namespace Xunit.Gherkin.Quick
 {
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
     [XunitTestCaseDiscoverer("Xunit.Gherkin.Quick.IgnoreMethodDiscoverer", "Xunit.Gherkin.Quick")]
```

Comment:
Now that I think about it, since you are inheriting from the regular Attribute class, you can remove this XunitTestCaseDiscoverer attribute too. That was needed to ensure it is ignored by the test explorer.
Also, remove the IgnoreMethodDiscoverer class too (cascading effect).

Reply:
good point - will fix that right away